packages/application-extension/src/index.ts

```diff
@@ -15,20 +15,19 @@ import {
   ProviderMock
 } from '@jupyterlab/docprovider';
 
-import { WebsocketProvider } from 'y-websocket';
-
 import { IMainMenu } from '@jupyterlab/mainmenu';
 
 import { ITranslator, TranslationManager } from '@jupyterlab/translation';
 
-const YJS_WEBSOCKET_URL = 'wss://demos.yjs.dev';
+import { WebrtcProvider } from 'y-webrtc';
 
-class WebSocketProvider extends WebsocketProvider implements IDocumentProvider {
+class WebRtcProvider extends WebrtcProvider implements IDocumentProvider {
   constructor(options: IDocumentProviderFactory.IOptions) {
-    super(YJS_WEBSOCKET_URL, options.guid, options.ymodel.ydoc);
+    super(options.guid, options.ymodel.ydoc);
+    this.awareness = options.ymodel.awareness;
```

Comment:
We could also make the signaling server(s) configurable via the settings / overrides.

Reply:
Did this happen? I think we should add this configuration sprocket before merging.